roles/fluentd_master/tasks/main.yml

```diff
@@ -39,8 +39,11 @@
     owner: 'td-agent'
     mode: 0444
 
-- name: "Pause before restarting td-agent and openshift-master, depending on the number of nodes."
-  pause: seconds={{ ( num_nodes|int < 3 ) | ternary(15, (num_nodes|int * 5)) }}
+- name: wait for etcd to start up
+  wait_for: port=4001 delay=10
+
+- name: wait for etcd peer to start up
+  wait_for: port=7001 delay=10
```

Comment:
Where is this service going to be co-located with?  4001/7001 are the etcd ports only on hosts that are running the embedded etcd, so it will exclude any deployments using an external etcd.